verifier/verifier.py

```
@@ -217,6 +219,8 @@ def verify(self, email):
                 lookup['message'] = "Internal Error"
             except smtplib.SMTPConnectError as err:
                 lookup['message'] = "Internal Error. Maybe blacklisted"
+            except smtplib.SMTPException as err:
```

Ruff (0.15.2)

[error] Local variable `err` is assigned to but never used

Remove assignment to unused variable `err`

(F841)